pkg/ttnpb/fieldmask_utils.go

```diff
@@ -317,3 +317,17 @@ func ApplySessionKeysFieldMask(dst, src *SessionKeys, paths ...string) (*Session
 func FilterGetSessionKeys(pb *SessionKeys, paths ...string) (*SessionKeys, error) {
 	return ApplySessionKeysFieldMask(nil, pb, AddImplicitSessionKeysGetFields(paths...)...)
 }
+
+// IncludeFields returns the given paths with the given search paths to include.
+func IncludeFields(paths []string, includePaths ...string) []string {
```

Comment:
How is this different from `AllowedFields()`?

Reply:
`AllowedFields` uses only full paths (`ContainsField`), while this one allows prefix paths (`HasAnyField`).

Reply:
`AllowedFields()` uses `ContainsField()` while `IncludeFields()` uses `HasAnyField()`

Reply:
Ah ok